test-assets/phpcs.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<ruleset>
+  <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,md,yml"/>
+  <file>web/sites/default</file>
+  <!-- Ignore generated assets -->
+  <rule ref="./vendor/drupal/coder/coder_sniffer/Drupal/ruleset.xml">
```

Comment:
Rather than using file references, install `dealerdirect/phpcodesniffer-composer-installer` together with phpcs, and simply refer to `Drupal`.

Reply:
Sadly we're not using `phpcodesniffer-composer-installer` much, though we really should.

Reply:
I am not sure how to refer to Drupal and I already spent quite some time on this task.
In my opinion this works fine for now. The rule set is copied from reloaddk.
Optionally we can create a task for it in the internal backlog
@xendk

Reply:
Like so: https://github.com/reload/action-phpcs/pull/6/commits/6d955aa787dc0318b05f82de3cdebf61fb39fd97

I still don't like the fact that there's no indication that phpcs is actually doing what we think it does.